Index: src/main/java/models/repository/BaseRepository.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/models/repository/BaseRepository.java b/src/main/java/models/repository/BaseRepository.java
--- a/src/main/java/models/repository/BaseRepository.java	
+++ b/src/main/java/models/repository/BaseRepository.java	
@@ -7,7 +7,7 @@
 public class BaseRepository {
     private static final String URL ="jdbc:mysql://localhost:3306/ig1_store"; // sửa lại tên của csdl
     private static final String USER ="root";// mặc định của mysql
-    private static final String PASS ="codegym";// do cài đặt khi cài đặt mysql
+    private static final String PASS ="Khanh@28082001";// do cài đặt khi cài đặt mysql
     public static Connection getConnectDB(){
         Connection connection = null;
         try {
